src/app/components/sponsor/sponsor.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, HostBinding, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'niz-sponsor',
+  template: `
+    <div class="relative">
+      <div
+        class="absolute -inset-1 rounded-lg bg-gradient-primary-inverse"
+      ></div>
+      <div class="relative rounded-lg bg-canvas p-4">
+        <div class="flex flex-row items-baseline">
+          <niz-inline-svg
+            class="h-5 w-5 flex-shrink-0 text-primary"
+            svgSource="assets/img/heart.svg"
+          ></niz-inline-svg>
+
+          <div class="ml-3">
+            <h2>Sponsor us</h2>
+            <p class="mt-2 text-lg font-medium">
+              <strong class="font-semibold"
+                >Did you find this post or any other useful?</strong
```

Comment:
Did you find this post ~~or any other~~ useful?

Reply:
Needs a sentence about what we do

Reply:
Add possibility to do a one time donation. Definitely more likely to happen when someone stumbles over this when googling around